k4TestFWCore/src/components/HelloWorldAlg.h

```diff
@@ -0,0 +1,35 @@
+#ifndef TESTFWCORE_HELLOWORLDALG
+#define TESTFWCORE_HELLOWORLDALG
+
+#pragma once 
+
+#include<string>
+
+// GAUDI
+#include "GaudiAlg/GaudiAlgorithm.h"
+#include "Gaudi/Property.h"
+
+
+class HelloWorldAlg : public GaudiAlgorithm {
+public:
+  explicit HelloWorldAlg(const std::string&, ISvcLocator*);
+  virtual ~HelloWorldAlg();
+  /**  Initialize.
+   *   @return status code
+   */
+  virtual StatusCode initialize() final;
+  /**  Execute.
+   *   @return status code
+   */
+  virtual StatusCode execute() final;
+  /**  Finalize.
+   *   @return status code
+   */
+  virtual StatusCode finalize() final;
+
+private:
+  // member variable 
+  std::string message = "Hello ";
```

Comment:
Maybe you can change this into a `Property` in order to show how to set thinks from the job options?

Reply:
yea, sounds good. done